### ISSUE TITLE
Lato Font Family

### ISSUE THREAD

**Issue by jspinella** [jatsby/jatsby-framework#1]
The Jatsby Framework provides the complete Lato font family (from 100 to 900) via jatsby.com.

As it stands, the lato.css file, which contains @font-face code for Lato by font weight and style (normal v. italic), does NOT include any src: local provisions. Regardless of the existence of Lato locally, it will always be imported from jatsby.com.

Obviously this isn't ideal for page-loading optimization and uses unnecessary bandwidth, however when src:local was used, issues were encountered in testing.

When the complete Lato offering from Google Fonts was installed locally (via SkyFonts), Coda would display a generic "character" square-shaped glyph for weights 400 and 700. When loading from jatsby.com was disabled, the issues expanded to include 500 and 600, which are not provided by Google Fonts (as a note, Google Fonts provides font files for Lato 100, 300, 400, 700 and 900).

I'm reaching out to Panic, since this seems like a Coda issue, however since Coda is vital to website testing for the Jatsby Framework, I've decided to "band-aid" the problem by, as mentioned above, removing all src:local declarations from lato.css for the time being.

Additionally, there are some issues to be worked out with 200 becoming 300 and 500 becoming 400 when Lato is enabled locally and lato.css is used to import the weights not offered by Google's Lato. This issue is peculiar, since 200 and 500 are not local (again, they aren't in Google's Lato package), and are not indicated as local by lato.css.

To sum up, the only 100% working solution is to simply leave src:local out of lato.css and to thereby not use any local instances of Lato. This is not a permanent solution of course, and will be addressed eventually. Priority is relatively low, but set to be fixed by Jatsby Framework v1.0 (the first release).
